index.src.html

```diff
@@ -1241,22 +1295,24 @@ <h4 algorithm id="construct-passwordcredential-form">
   </h4>
 
   To <dfn abstract-op>Create a `PasswordCredential` from an `HTMLFormElement`</dfn>, given an
-  {{HTMLFormElement}} (|form|), run these steps.
+  {{HTMLFormElement}} (|form|) and an [=origin=] (|origin|), run these steps.
   
   Note: [[#examples-post-signin]] and [[#examples-change-password]] provide examples of the intended
   usage.
 
   <ol class="algorithm">
     1.  Let |data| be a new {{PasswordCredentialData}} dictionary.
 
-    2.  Let |formData| be the result of executing the {{FormData}} constructor
+    2.  Set |data|'s {{PasswordCredentialData/origin}} member's value to |origin|'s value.
```

Comment:
Rather than publicly add `origin` to `PasswordCredentialData`, I'd suggest just passing it to the "Create a `PasswordCredential`" algorithm you adjust below. That is, drop this new step 2, change the existing step 6 to something like the following:

```
6.  Let |c| be the result of executing <a abstract-op>Create a `PasswordCredential` from
    `PasswordCredentialData`</a> |data| and |origin|.
```

And note that the relevant algorithm takes an origin.